src/main/webapp/app/shared/link-preview/components/link-preview-container/link-preview-container.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { LinkPreview, LinkPreviewService } from 'app/shared/link-preview/services/link-preview.service';
+import { Link, LinkifyService } from 'app/shared/link-preview/services/linkify.service';
+
+@Component({
+    selector: 'jhi-link-preview-container',
+    templateUrl: './link-preview-container.component.html',
+    styleUrls: ['./link-preview-container.component.scss'],
+})
+export class LinkPreviewContainerComponent implements OnInit {
+    // to forward
+    @Input() color = 'primary'; // accent | warn
```

Comment:
Are accent and warn the other valid values for this input?
Why not explicitly put the type here with the three of them

Reply:
I removed the `color ` field as i am not using at all.